.github/workflows/mega-linter.yml

```diff
@@ -0,0 +1,188 @@
+# MegaLinter GitHub Action configuration file
+# More info at https://megalinter.io
+---
+name: MegaLinter
+
+# Trigger mega-linter at every push. Action will also be visible from
+# Pull Requests to main
+on:
+  push:
```

Comment:
We don't really need this, running the linters only on a PR is fine. 

As a side effect it now runs twice, see the statuses reported on the PR.